common/achievement/get.ts

```diff
@@ -192,22 +188,31 @@ const assembleAchievementData = async (userAchievements: achievements_with_templ
                               );
                           })
                           .reduce((a, b) => a + b, 0);
+            if (currentValue < maxValue && userAchievements[currentAchievementIndex].achievedAt) {
+                await prisma.user_achievement.update({
+                    where: { id: userAchievements[currentAchievementIndex].id },
+                    data: { achievedAt: null, isSeen: false },
+                });
```

Comment:
I don't think that this belongs here.
It would mean that we have an invalid state for some reason, which shouldn't be fixed while `assembling data`.

When does it happen?

Reply:
Imagine you have a streak of 25 consecutive meetings joined.
Now you do not join any more meetings. You do not fire any more events "match_meeting_joined" or what ever the action name is.
The streak would never be resolved into being 0 because we never reevaluate. We offered a cronjob, that was already implemented, but that was not wanted either. So we have to update streaks here.